Tests/TestFuncs/TestBasicFuncs/FixtureNonSimpleFuncs.swift

```diff
@@ -370,7 +370,7 @@ import Foundation
 
 /// \(String.mockAnnotation)
 protocol NonSimpleFuncs {
-func pass<T>(handler: @autoclosure () -> Int) rethrows -> T
+func pass<T>(handler: @autoclosure () -> Int) throws -> T
```

Comment:
This is bug fix, right?

Reply:
Sure. The original fixture code is incorrect.